receiver/splunksearchapireceiver/model.go

```diff
@@ -58,3 +58,8 @@ type SearchResultsResponse struct {
 		Time string `json:"_time"`
 	} `json:"results"`
 }
+
+// EventRecord struct stores the offset of the last event exported successfully
+type EventRecord struct {
+	Offset int `json:"offset"`
```

Comment:
Curious last time I've used a storage extension; we've used a timestamp to manage the last processed event time... Do you think that'd be more bulletproof than using the paginated offset?

Reply:
Using the offset instead of the timestamp was a piece of feedback from Dan on the spec. Looking back at his comment, he thought it would be easier to use the offset. Whether it's as bulletproof is a good question, but I assumed he wouldn't sacrifice the reliability of the receiver for ease. Curious to hear more of your perspective.